src/app/modules/slideshow/components/slideshow/slideshow.component.html

```diff
@@ -19,6 +19,7 @@ <h1>{{ page.title }}</h1>
           </mat-card-header>
           <mat-card-content>
             <img [src]="img.url" [alt]="img.description"/>
+            <p>{{ img.subtitle }}</p>
```

Comment:
Wenn wir hier mit dem Material Krams konsistent sein wollen, gehört der Subtitle in einen `<mat-card-subtitle>` und nicht unter das Bild in ein Paragraphen.

Reply:
im letzten Commit auf `text` geändert, da es eher eine Bildunterschrift als ein Untertitel ist.

Reply:
Da in den meisten Fällen aber konkrete Mineralbilder sind, könnte man auch beides haben, `subtitle` (analog zur Galerie) und `text` für ausführlichere Bildunterschriften.